notebooks/NIRISS_WFSS_advanced/01_niriss_wfss_image2_image3.ipynb

```diff
@@ -0,0 +1,1025 @@
+    "**Data**: JWST/NIRISS images and spectra from program 2079.<br>\n",
+    "**Tools**: astropy, glob, jdaviz, json, jwst, matplotlib, numpy, os, pandas, warnings<br>\n",
+    "**Cross-instrument**: NIRISS<br>\n",
+    "\n",
+    "**Content**\n",
+    "- [Imports & Data Setup](#imports)\n",
+    "- [Default Imaging Pipeline Run](#default)\n",
+    "  - [Image2](#default_image2)\n",
+    "  - [Image3](#default_image3)\n",
+    "  - [Inspecting Default Results](#view_default)\n",
+    "- [Custom Imaging Pipeline Run](#custom)\n",
+    "  - [Image3](#custom_image3)\n",
+    "  - [Inspecting Custom Results](#view_custom)\n",
+    "- [Refining the Source Catalog Further](#source_cat)\n",
+    "  - [Matching Source IDs Across Catalogs](#match_sources) \n",
+    "  - [Manually Editing the Source Catalog](#manual_cat)\n",
+    "\n",
+    "**Author**: Rachel Plesha (rplesha@stsci.edu), Camilla Pacifici (cpacifici@stsci.edu), JWebbinar notebooks.<br>\n",
+    "**First Published**: January 2024"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7fdaf1a1-57e5-4239-b82e-6552261d8d82",
+   "metadata": {},
+   "source": [
+    "<a id='imports'></a>\n",
+    "## Imports & Data Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "028352f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import glob\n",
+    "import json\n",
+    "import warnings\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import astropy.units as u\n",
+    "from astropy.io import fits\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from astropy.table import Table\n",
+    "\n",
+    "from jdaviz import Imviz\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import jwst\n",
+    "from jwst.pipeline import Image2Pipeline\n",
+    "from jwst.pipeline import Image3Pipeline"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "42888b91-e010-4ed2-b101-f586668a2235",
+   "metadata": {},
+   "source": [
+    "# Update the CRDS path to your local directory\n",
+    "%env CRDS_PATH=<local cache> (/grp/crds/jwst/)\n",
+    "%env CRDS_SERVER_URL=https://jwst-crds.stsci.edu"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb035a4d-e63c-4125-b083-462e3353e49f",
+   "metadata": {},
+   "source": [
+    "Check what version of the JWST pipeline you are using. To see what the latest version of the pipeline is available, check <a href=\"https://github.com/spacetelescope/jwst#software-vs-dms-build-version-map\">here</a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae10a560",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('jwst:', jwst.__version__)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e2ecc213-479a-422c-a0f8-8608309b845c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_dir = 'data'\n",
```

Comment:
I think this needs a bit of explanation of how files should be stored in this directory. Like MAST downloaded them? All rate and asn in data? Rate and asn in separate directories inside data?